src/cast_core/src/Loader.cpp

```diff
@@ -117,6 +118,9 @@ void genny::actor::Loader::run() {
                 auto collectionName = "Collection" + std::to_string(i);
                 auto collection = config->database[collectionName];
                 // Insert the documents
+                BOOST_LOG_TRIVIAL(info) << "Starting to insert: "
```

Comment:
This was from Ger's work earlier. It doesn't seem unreasonable to have the start and end logged, but I can pull it if it's distracting.

Reply:
Sounds good, it sounds like a reasonable log to have